arithmetization/src/main/java/net/consensys/linea/zktracer/module/rlptxn/RlpTxn.java

```diff
@@ -1214,9 +1214,8 @@ public List<ColumnHeader> columnsHeaders() {
   public void commit(List<MappedByteBuffer> buffers) {
     final Trace trace = new Trace(buffers);
     int absTxNum = 0;
-    for (RlpTxnChunk chunk : this.chunkList) {
-      absTxNum += 1;
-      traceChunk(chunk, absTxNum, trace);
+    for (RlpTxnChunk chunk : operations.getAll()) {
+      traceChunk(chunk, ++absTxNum, trace);
```

Comment:
We should get rid of the `chunk` terminology while we're at it (at least for modules other than ROM related ones) in favour of `operation` / `operations` / `traceOperations` / ...

Reply:
done